### ISSUE TITLE
Non-overlapping data

### ISSUE THREAD

**Issue by sebftw** [gulvarol/surreal#32]
Is the pose parameters for the sequences with no overlapping frames available?

I tried to work around this by making my own algorithm to detect overlaps and then cut and splice the clips into non-overlapping sequences. Running that on run1 of the test and training data I found 2666 non-overlapping sequences (703 in test, 1963 train), does that seem about right?

**Comment by sebftw** [comment]
Ah sorry, looking through the paper I see there are indeed 703 sequences in the test set.

**Comment by gulvarol** [comment]
Do you mean MoCap sequences or SURREAL sequences? The pose parameters for the non-overlapping MoCap sequences are found in smpl_data.npz file. All synthetic videos are rendered with overlapping windows.

**Comment by sebftw** [comment]
@gulvarol I see.
That was just what I needed. Thanks!
